SocialMediaDB.sql

```diff
@@ -64,3 +64,87 @@ CREATE TABLE PROFILE (
     sexes ENUM('Male', 'Female', 'Others', 'PreferNotToSay'),
     CONSTRAINT OWNS_PROFILE FOREIGN KEY (user_id) REFERENCES USER (user_id) on DELETE CASCADE on UPDATE CASCADE
 ) ENGINE = InnoDB DEFAULT CHARSET = utf8;
+
+CREATE TABLE EDUCATION (
+    user_id INT,
+    education VARCHAR(100),
+    PRIMARY KEY (user_id, education),
+    FOREIGN KEY (user_id) REFERENCES PROFILE(user_id) ON DELETE CASCADE ON UPDATE CASCADE
+);
+
+CREATE TABLE PAGE (
+    page_id int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    page_name varchar(50) NOT NULL,
+    owner_id int,
```

Comment:
I think page name can have more than 50 characters, but we can fix these limits later after we are done with this SQL dump.